_posts/kate_jared_wedding_vows.md

```diff
@@ -0,0 +1,59 @@
+### Kate Jared Wedding Ceremony
+
+## Introduction
+
+Dearly beloved, Friends and family. We are gathered here today to observer and participate in the ageless tradition of marriage. We're here to witness and support Kate and Jared. Today is a ceremony to celebrate a union of two individuals. A union that will enhance the lives of them as individuals and the world around them as a couple.
```

Comment:
Friends & Family. Today is a celebration. A celebration of love, of commitment, of friendship, of family, and of two people who are in it for forever.